api/inventorypb/agents.proto

```diff
@@ -325,6 +325,28 @@ message RDSExporter {
   bool enhanced_metrics_disabled = 10;
 }
 
+// ExternalExporter runs on any Node.
+message ExternalExporter {
+  // Unique randomly generated instance identifier.
+  string agent_id = 1;
+  // Node identifier where this instance runs.
+  string runs_on_node_id = 2;
+  // Desired Agent status: enabled (false) or disabled (true).
+  bool disabled = 3;
```

Comment:
What does it mean for external exporter?

Reply:
Will we scrape external exporter or not. Good point, I ignored it in pmm-managed.

Reply:
For other exporters, it means that it will not be started by pmm-agent. That's why it is called desired – it may be running now, but pmm-agent will stop it soon.

I think we should remove the whole field for external exporters.

Reply:
What if user would like to pause collecting metrics for some time? We let him to do it for other exporters. I can fix comment to something like `Desired metrics collection status`

Reply:
ok, let's keep the name, but make a comment/documentation distinctively different

Reply:
@BupycHuk ^^^

Reply:
Updated